README.md

```diff
@@ -22,8 +22,9 @@ The project runs on Python 3.
 1. Create a virtual environment:
 `virtualenv venv --python=python3`
 
-2. Activate the virtual environment:
-`source ./venv/bin/activate`
+2. Activate the virtual environment:<br />
```

Comment:
Can you remove this part as this is not part of the issue? Also there is a PR for this already https://github.com/anitab-org/mentorship-backend/pull/215 You can even help review if you are using the Windows environment